builtin/trap_osh.py

```diff
@@ -152,6 +152,16 @@ def ThisProcessHasTraps(self):
         return len(self.traps) != 0 or len(self.hooks) != 0
 
 
+def _IsUnsignedInteger(s):
+    # type: (str) -> bool
+
+    try:
+        if int(s) >= 0:
```

Comment:
Minor thing - for user facing strings, we use `mops.FromStr(s)` -- this is because when mycpp translates it, `int()` operates on C `int`, while `mops.FromStr(s)` is the 64-bit type, which may become big integers


And I would perhaps wrap that in its own `try/except` so it's clear where ValueError is raised

Reply:
I *think* I got this right; I'm not at all familiar with mops, but I made some guesses until the tests passed again.